index.html

```diff
@@ -9,44 +9,49 @@
 
 </head>
 <body>
-    <section class="section-one">
         <header>
     
-            <div class="header-containt">
-    
-                <nav>
-                    <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
-                    </ul>
-    
-                </nav>
-    
-            </div>
+           <!-- Add a navigation icon -->
+          <button class="nav-icon">
+            <i class="fas fa-bars"></i>
+          </button>
```

Comment:
Very well done by moving the header outside the section tag as requested 👍🏼  , Please wrap this button in a nav tag as it is for navigation purpose this was requested by the previous reviewer as well